README.md

```diff
@@ -1,2 +1,108 @@
-# service-catalog-tester
-Contains reproducible test-cases for service-catalog.
+# Service Catalog Tester
+
+## Overview
+
+This project contains reproducible test-cases for service-catalog.
+
+## Prerequisites
+
+To set up the project, use these tools:
+* Version 1.9 or higher of [Go](https://golang.org/dl/)
+* The latest version of [Docker](https://www.docker.com/)
+* The latest version of [Dep](https://github.com/golang/dep)
+
+## Usage
+
+This section explains how to use the Etcd Backup tool.
+
+### Use environment variables
+Use the following environment variables to configure the application:
+
+| Name | Required | Default | Description |
+|-----|---------|--------|------------|
+|**APP_PORT** | NO | `8080` | The port on which the HTTP server listens |
+| **APP_LOGGER_LEVEL** | No | `info` | Show detailed logs in the application. |
+| **APP_KUBECONFIG_PATH** | No |  | The path to the `kubeconfig` file that you need to run an application outside of the cluster. |
+| **APP_SLACK_CLIENT_CHANNEL_ID** | Yes |  | The Slack channel where notification are posted. |
+| **APP_SLACK_CLIENT_WEBHOOK_URL** | Yes |  | The Slack Webhook URL. |
+| **APP_SLACK_CLIENT_TOKEN** | Yes |  | The Slack token used as the key to messages on Slack channel. |
+| **APP_OBSERVABLE_NAMESPACE** | Yes |  | The name of the Namespace where observed Deployments are installed. |
+| **APP_OBSERVABLE_DEPLOYMENTS_NAMES** | Yes |  | The names of Deployment which need to observed. Multiple Deployments names should be separated by comma. |
```

Comment:
`The names of Deployments which need to be observed.` or `The names of Deployments you want to observe.` ? The second one would be better from the perspective of our guidelines but I'm not sure whether it's technically correct.